matter/si91x/siwx917/BRD4325B/config/em_core_generic.h

```diff
@@ -0,0 +1,167 @@
+/***************************************************************************/ /**
```

Comment:
nit: removing the `/  /` at the end of the line here and in some function briefs of this file and `emlib_core_debug_config.h` and then applying the styler should fix the wrong indentation done by the styler.

Reply:
done